src/app/components/feed/CreateFeed.js

```diff
@@ -0,0 +1,218 @@
+import React from 'react';
+import { FormattedMessage, FormattedHTMLMessage } from 'react-intl';
+import Checkbox from '@material-ui/core/Checkbox';
+import ListIcon from '@material-ui/icons/List';
+import styles from './CreateFeed.module.css';
+import ExternalLink from '../ExternalLink';
+import Select from '../cds/inputs/Select';
+import SwitchComponent from '../cds/inputs/SwitchComponent';
+import TextArea from '../cds/inputs/TextArea';
+import TextField from '../cds/inputs/TextField';
+import SchoolIcon from '../../icons/school.svg';
+import CorporateFareIcon from '../../icons/corporate_fare.svg';
+import OpenSourceIcon from '../../icons/open_source.svg';
+
+/*
+  TODO
+  - Review with Brian
+    - License icons
+    - Verify proper casing
+      - Review copy? (license ... can do with your code)
+    - Borderless cards
+    - Shared feed overline title
+*/
+
+const LicenseOption = ({
+  icon,
+  title,
+  description,
+  url,
+}) => (
+  <div className={styles.licenseOption}>
+    <Checkbox />
+    <div className={styles.licenseOptionIcon}>
+      {icon}
+    </div>
+    <div>
+      <div className="typography-subtitle2">
+        {title}
+      </div>
+      {description}
+      {url && (
+        <>
+          <span> </span> { /* white space before license link */}
```

Comment:
Suggestion: change this to `&nbsp;` (non-breaking space) instead of ` ` -- I think that way we won't even need a comment here since it will be evident that the space is intended.